Source/lucene-query.peg

```diff
@@ -169,13 +169,13 @@ name_term
     }
 
 unquoted_term
-  = term:(([^: \\\t\r\n\f\{\}\(\)/^~\[\]\*] / '\\' [:-\\\(\)^~\*])+ "")
+  = term:(([^: \\\t\r\n\f\{\}\(\)/^~\[\]] / '\\' [:-\\\(\)^~])+ "")
     {
         term
     }
 
 range_unquoted_term
-  = term:((([^: \\\.\t\r\n\f\{\}\(\)"^~\[\]\*] / '\\' [:-\\\(\)^~\*])+ "") / '*')
+  = term:((([^: \\\.\t\r\n\f\{\}\(\)"^~\[\]] / '\\' [:-\\\(\)^~])+ "") / '*')
     {

```

Comment:
I'm not sure why wildcards are in this exclude regex when it's a valid character. I don't know what the `/ '*'` is doing? Ideas?